packages/telestion-client-common/src/components/pages/login-page/login-page.tsx

```diff
@@ -40,11 +40,19 @@ export interface LoginPageProps {
  * - {@link LoginDescription}
  * - {@link LoginForm}
  *
+ * * **Attention**
+ *
+ * If you create your own login page, please pass through
+ * the routing information so the pages component read it
+ * and render the page the right way:
+ * `MyLoginPage.routing = TCLoginPage.routing;`
```

Comment:
```suggestion
 * If you create your own login page, please pass
 * the routing information so that the pages component read it
 * and render the page the right way:
 * `MyLoginPage.routing = TCLoginPage.routing;`
```